src/freenet/clients/http/QueueToadlet.java

```diff
@@ -1353,7 +1355,17 @@ else if(secondFinalized && !firstFinalized)
 							result = Fields.compare(firstProgress, secondProgress);
 						}
 					} else if (sortBy.equals("lastActivity")) {
-						result = Fields.compare(firstRequest.getLastActivity(), secondRequest.getLastActivity());
+                        Date a = firstRequest.getLastActivity();
+                        Date b = secondRequest.getLastActivity();
+                        if(a == null) a = new Date(0);
```

Comment:
The [coding standards require](https://google-styleguide.googlecode.com/svn/trunk/javaguide.html#s4.6.2-horizontal-whitespace) a space between the `if` and `(`.